docs/photoCredits.txt

```diff
@@ -0,0 +1,17 @@
+PHOTO CREDITS:

```

Comment:
The photoCredits.txt file should be named PHOTO-CREDITS instead, or PHOTO-CREDITS.md if you want to format it with markdown.  This is consistent with the convention naming project meta files, like LICENSE, README, AUTHORS, TODO, etc.